go/genesis/api/api.go

```diff
@@ -80,6 +81,54 @@ func (d *Document) WriteFileJSON(filename string) error {
 	return nil
 }
 
+// SanityCheck does basic sanity checking on the contents of the genesis document.
+func (d *Document) SanityCheck() error {
+	if d.Height < 0 {
+		return fmt.Errorf("genesis: sanity check failed: height must be >= 0")
+	}
+
+	// TODO: Maybe add some epsilon here to account for network latency and time drift?
```

Comment:
Don't think this is required.  The genesis document will be generated a good amount of time prior to the sanity check being applied.